File: GerenciaMusic360/ClientApp/app/shared/models/template-task-document-detail.ts
export interface TemplateTaskDocumentDetail {
    id: number,
    name: string,
    templateTaskDocumentId: number,
    required: boolean,
    position: number,
    usersAuthorize: number[],
    estimatedDateVerficationString: string,
    isPermanent: boolean,
    projectId: number,
    entityId: number,
}

File: GerenciaMusic360.Entities/TemplateTaskDocumentDetail.cs
using System.Collections.Generic;

namespace GerenciaMusic360.Entities
{
    public partial class TemplateTaskDocumentDetail
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public short Position { get; set; }
        public int TemplateTaskDocumentId { get; set; }
        public string TemplateTaskDocumentName { get; set; }
        public List<long> UsersAuthorize { get; set; }
        public string EstimatedDateVerficationString { get; set; }
        public bool Required { get; set; }
        public bool IsPermanent { get; set; }
        public int ProjectId { get; set; }
        public short EntityId { get; set; }

    }
}
